class.jetpack-jitm.php

```diff
@@ -56,490 +55,275 @@ function get_emblem()
-					</p>
-				<?php elseif ( $manage_active ) : ?>
-					<p>
-						<?php esc_html_e( 'Save time with auto updates on WordPress.com', 'jetpack' ); ?>
-					</p>
-				<?php endif; // manage inactive
-				?>
-				<p class="show-after-enable <?php echo $manage_active ? '' : 'hide'; ?>">
-					<a href="<?php echo esc_url( 'https://wordpress.com/plugins/' . $normalized_site_url ); ?>" target="_blank" title="<?php esc_attr_e( 'Go to WordPress.com to enable auto-updates for plugins', 'jetpack' ); ?>" data-module="manage-pi" class="button button-jetpack launch show-after-enable"><?php if ( ! $manage_active ) : ?><?php esc_html_e( 'Enable auto-updates on WordPress.com', 'jetpack' ); ?><?php elseif ( $manage_active ) : ?><?php esc_html_e( 'Enable auto-updates', 'jetpack' ); ?><?php endif; // manage inactive ?></a>
-				</p>
-			</div>
-			<?php
-			//jitm is being viewed, track it
-			$jetpack = Jetpack::init();
-			$jetpack->stat( 'jitm', 'manage-pi-viewed-' . JETPACK__VERSION );
-			$jetpack->do_stats( 'server_side' );
-		endif; // manage inactive
+		return $content;
 	}
 
 	/**
-	 * Display message in editor prompting user to compose entry in WordPress.com.
+	 * A special filter for WooCommerce Call To Action button
 	 *
-	 * @since 3.8.2
+	 * @param $CTA string The existing CTA
+	 *
+	 * @return string The new CTA
 	 */
-	function editor_msg() {
-		global $typenow;
-		if ( current_user_can( 'manage_options' ) ) {
-			$normalized_site_url = Jetpack::build_raw_urls( get_home_url() );
-			$editor_dismissed = isset( self::$jetpack_hide_jitm['editor'] );
-			if ( ! $editor_dismissed ) :
-			?>
-			<div class="jp-jitm">
-				<a href="#"  data-module="editor" class="dismiss"><span class="genericon genericon-close"></span></a>
-				<?php echo self::get_emblem(); ?>
-				<p class="msg">
-					<?php esc_html_e( 'Try the brand new editor.', 'jetpack' ); ?>
-				</p>
-				<p>
-					<a href="<?php echo esc_url( 'https://wordpress.com/' . $typenow . '/' . $normalized_site_url ); ?>" target="_blank" title="<?php esc_attr_e( 'Write on WordPress.com', 'jetpack' ); ?>" data-module="editor" class="button button-jetpack launch show-after-enable"><?php esc_html_e( 'Write on WordPress.com', 'jetpack' ); ?></a>
-				</p>
-			</div>
-			<?php
-			//jitm is being viewed, track it
-			$jetpack = Jetpack::init();
-			$jetpack->stat( 'jitm', 'editor-viewed-' . JETPACK__VERSION );
-			$jetpack->do_stats( 'server_side' );
-			endif; // manage or editor inactive
-		}
+	static function jitm_jetpack_woo_services_install( $CTA ) {
+		return wp_nonce_url( add_query_arg( array( 'wc-services-action' => 'install' ), '/wp-admin/admin.php?page=wc-settings' ), 'wc-services-install' );
```

Comment:
Not sure if it's necessary, but this could be rewritten to introduce `admin_url()` 
```php
return wp_nonce_url( add_query_arg( array(
    'wc-services-action' => 'install'
), admin_url( 'admin.php?page=wc-settings' ) ), 'wc-services-install' );
```